### PR TITLE
add info on the 'change row display' settings in the ide

### PR DESCRIPTION
**Pull request by mirnawong1**
adding this new feature per @StefanDBTLabs 's update.

### CODE REVIEW

File: website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md
Comment (StefanDBTLabs) [inline website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md]: 👍

**Review by StefanDBTLabs — APPROVED** [review]

**Comment by mirnawong1** [comment]
Using this pr to add upcoming changes provided by Stefan.

Please hold off from merging until i add this.

File: website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md
Comment (runleonarun) [inline website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md]: I think we can remove the parentheses.
```suggestion
The Invocation History Drawer stores information on dbt invocations in the IDE. When you invoke a command, like executing a dbt command such as `dbt run`, the associated logs are displayed in the Invocation History Drawer. 
```

File: website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md
Comment (runleonarun) [inline website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md]: This looks good in output, but curious of it needs the space in the image title?

File: website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md
Comment (runleonarun) [inline website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md]:
```suggestion
   * **500-row limit:** To prevent the IDE from returning too much data and causing browser problems, dbt automatically sets a 500-row limit when using the **Preview Button**. You can modify this by adding `limit your_number` at the end of your SQL statement. For example, `SELECT * FROM` table `limit 100` will return up to 100 rows. Remember that you must write the `limit your_number` explicitly and cannot derive it from a macro.
```

File: website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md
Comment (runleonarun) [inline website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md]:
```suggestion
   * **Change row limit default:** In dbt version 1.6 or higher, you have the ability to change the default limit of 500 rows shown in the **Results** tab when you run a query. To adjust the setting you can click on **Change row display** next to the displayed rows. Keep in mind that you can't set it higher than 10,000 rows. If you refresh the page or close your development session, the default limit will go back to 500 rows.
```

File: website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md
Comment (runleonarun) [inline website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md]:
```suggestion
   * **Specify records returned:** The IDE also supports `SELECT TOP #`, which specifies the number of records to return.
```

File: website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md
Comment (runleonarun) [inline website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md:86]: @mirnawong1 This step has a lot of notes under it and I worry they will get lost. I suggested adding labels to each note to make them easily scannable, but I wonder if we should add section like "Default row limits in IDE" (or something) and then do this:
> To prevent the IDE from returning too much data and causing browser problems, dbt automatically sets a 500-row limit to queries executed using the Preview Button. You can change this default limit in dbt version 1.6 and higher. Learn more by referring to "Default row limits in IDE." Alternatively, you can specify the number of records to return using `SELECT TOP #` Learn more by referring to ???

**Review by runleonarun — COMMENTED** [review]
Hey Mirna! Suggesting adding a section for default row limit details. OR adding headings to paragraphs. Whichever feels like it would be more helpful top users.

File: website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md
Comment (mirnawong1) [inline website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md]:
```suggestion
<Lightbox src="/img/docs/dbt-cloud/cloud-ide/version-control-options-menu.png" width="30%" title="Right-click edited files to access Version Control Options menu"/>
```

File: website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md
Comment (mirnawong1) [reply website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md:86]: ah yes great call @runleonarun ! ok i've added a toggle to have it stand out more and also make it clear for users who intend on using this behavior -- what do you think?
Screenshot 2023-08-14 at 09 36 42
Screenshot 2023-08-14 at 09 36 47

Comment (runleonarun) [reply website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md:86]: Thanks @mirnawong1!! That feels much easier to parse quickly! Do you know if details tab content show up in algolia searches? For example, if someone searches the error message or "row limits in IDE."

Comment (mirnawong1) [reply website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md:86]: no worries! i think it does but let me verify with the front end team on this. i also think i should link this section in the ide [preview and build](https://deploy-preview-3904--docs-getdbt-com.netlify.app/docs/cloud/dbt-cloud-ide/develop-in-the-cloud)' section to help users navigate better.

**Review by runleonarun — APPROVED** [review]
Approving so you can merge, but left one non-blocking question about search results for this content, but feel free to merge and iterate if you'd like.

**Comment by mirnawong1** [comment]
merging this and front end team working on algolia crawl question!

**Pull request merged**
